website/docs/routes/study.mdx

```diff
@@ -689,3 +689,7 @@
   | 新疆维吾尔自治区 | 65   |
   | 台湾             | 71   |
 </Route>
+
+## Howtoforge Linux Tutorials
```

Comment:
Missing level 3 heading.
docs: https://docs.rsshub.app/joinus/new-rss/add-docs#other-things-to-keep-in-mind